### PR TITLE
Toolpath, segments, etc.

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: algorithm/cgal/fitPlaneToPoints.js [hunk 1]
```diff
@@ -11,8 +11,13 @@ export const fitPlaneToPoints = (points) => {
     const plane = [0, 0, 1, 0];
     c.FitPlaneToPoints(
       (triples) => {
+        let count = 0;
         for (const [x, y, z] of points) {
           c.addDoubleTriple(triples, x, y, z);
+          if (count++ > 1000) {
+            // We're blowing the stack when this is too high.
```
Comment (BarbourSmith) [inline algorithm/cgal/fitPlaneToPoints.js:18]: 😛 

Is it really as low as 1000?

File: geometry/tagged/computeToolpath.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+import { outline } from './outline.js';
+import { section } from './section.js';
+import { taggedToolpath } from './taggedToolpath.js';
+import { toConcreteGeometry } from './toConcreteGeometry.js';
+
+const X = 0;
+const Y = 1;
+const Z = 2;
+
+export const computeToolpath = (geometry, toolDiameter = 1, jumpHeight = 1) => {
```
Comment (BarbourSmith) [inline geometry/tagged/computeToolpath.js:17]: This is impressively compact for everything that it's doing

**Review by BarbourSmith — APPROVED** [review]
It all looks good to me 😀 

Some of the .pngs in plotter aren't showing the same thing that they were before, but it doesn't look wrong. Are they different because toolpaths aren't in soup so they don't show up in .png?

File: geometry/tagged/computeToolpath.js (hunk 2 above)
Comment (pentacular) [reply geometry/tagged/computeToolpath.js:17]: Yeah, but it's also doing things very inefficiently. :)

I suspect it'll get a bit larger.

**Pull request merged**